context-windows/src/context.ps1

```diff
@@ -1343,11 +1343,16 @@ If ( !(Test-Path "$ctxDir") ) {
     mkdir "$ctxDir"
 }
 
-# Move old logfile away - so we have a current log containing the output of the last boot
+# Delete .old log file (if exist) - simple rotation
+If (Test-Path "$ctxDir\opennebula-context-old.log") {
+    Remove-Item "$ctxDir\opennebula-context-old.log"
+}
+
+# Move last logfile away - so we have a current log containing the output of the last boot
 If ( Test-Path "$ctxDir\opennebula-context.log" ) {
     mv "$ctxDir\opennebula-context.log" "$ctxDir\opennebula-context-old.log"

```

Comment:
I think the change could be simplified by adding **-Force** parameter to **mv** (Move-Item) call.
[MS Docs for Move-Item](https://learn.microsoft.com/en-us/powershell/module/microsoft.powershell.management/move-item?view=powershell-5.1#-force)

Reply:
I thought about using -Force too, but decided to add few lines of code, so it's now seems a bit more clear of what's going on there

Reply:
Yeah. It's more readable that way. Functionally should be the same.